docs/docs/docs/01-core/admin/04-hooks/use-admin-role-permissions.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: useAdminRolePermissions
+description: API reference for the useAdminRolePermissions hook
+tags:
+  - admin
+  - hooks
+  - roles
+  - permissions
+---
+
+An abstraction around `react-query`'s `useQuery` hook. It can be used to fetch permissions for one admin role.
```

Comment:
Can you add why we need it, i.e. in what circumstances it might be useful, because if we only use it in one place, maybe there's no point abstracting it (i've not checked)

Reply:
We use it in two places at the moment (Create and Edit view for roles). I agree there is not much value in the hook at the moment and it will probably go away with route level data-fetching, but you never know?